src/hitchhiker/tree/node.cljc

```diff
@@ -21,10 +21,13 @@
   (-lookup [node k] "Returns the child node which contains the given key"))
 
 ;; marker protocols
-(defprotocol IDataNode)
-(defprotocol IIndexNode)
-(defprotocol IResolved)
+(defprotocol IDataNode
+  (-foo1 [node] "Does nothing"))
```

Comment:
this supposed to be like that?

Reply:
Yes, you can't have them empty in cljs. We can of course improve the naming. This was done as a quick fix for cljs compilation.